src/Interpreters/executeDDLQueryOnCluster.cpp

```diff
@@ -49,7 +49,7 @@ static ZooKeeperRetriesInfo getRetriesInfo()
     );
 }
 
-bool isSupportedAlterType(int type)
+bool isSupportedAlterTypeForDDLQuery(int type)
```

Comment:
If you rename this function, then please rename it to something like `isSupportedAlterTypeForDistributedDDLQuery` or `isSupportedAlterTypeForOnClusterDDLQuery`. Because `isSupportedAlterTypeForDDLQuery` is not anyhow better than `isSupportedAlterType`